src/app/shared/case-editor/case-edit-submit.html

```diff
@@ -65,20 +65,22 @@ <h2 class="plain-message">Check the information below carefully. This will form
       </tbody>
     </table>
   </ng-container>
-  <fieldset id="fieldset-event" formGroupName="event" *ngIf="!isSolicitor()">
-    <div class="form-group">
-      <label for="field-trigger-summary" class="form-label">
-        Event summary (optional)
-        <span class="form-hint">A few words describing the purpose of the event.</span>
-      </label>
-      <input type="text" id="field-trigger-summary" class="form-control" formControlName="summary" maxlength="1024">
-    </div>
-    <div class="form-group">
-      <label for="field-trigger-description" class="form-label">Event description (optional)</label>
-      <textarea id="field-trigger-description" class="form-control" formControlName="description"
-                maxlength="65536"></textarea>
-    </div>
-  </fieldset>
+  <ng-container *ngIf="showEventNotes()">
```

Comment:
I think we should to add a unit test for this template to verify the this whole block is not displayed if showEventNotes() if false

Reply:
Done.